presto-teradata-functions/src/test/java/com/facebook/presto/teradata/functions/TestTeradataDateFunctions.java

```diff
@@ -152,7 +147,7 @@ private void assertDate(String projection, int year, int month, int day)
         assertFunction(
                 projection,
                 DateType.DATE,
-                sqlDate(new DateTime(year, month, day, 0, 0, DateTimeZone.UTC)));
+                new SqlDate(toIntExact(LocalDate.of(year, month, day).toEpochDay())));
```

Comment:
Shouldn't commit message be:
`Fix TestTeradataDateFunctions to work with non-legacy timestamps and in zones with negative offset`?

Reply:
I don't think so.